src/lib.rs

```diff
@@ -40,8 +40,10 @@ impl<T> WithLock<T> {
 	///
 	/// let a = WithLock::<i64>::new(Mutex::new(2));
 	/// let b = WithLock::<i64>::new(Mutex::new(3));
-	/// let a_lock = a.with_lock(|s| *s);
-	/// let b_lock = b.with_lock(|s| *s);
+	/// let action_and_get = |s: &mut i32| *s;
```

Comment:
```suggestion
	/// let action_and_get = |s: &mut i64| *s;
```
It's actually an i64 :)